src/panels/lovelace/components/hui-action-editor.ts

```diff
@@ -115,51 +240,142 @@ export class HuiActionEditor extends LitElement {
             ></paper-input>
           `
         : ""}
-      ${this.config?.action === "call-service"
+      ${this._config?.action === "call-service"
         ? html`
             <ha-service-picker
               .hass=${this.hass}
               .value=${this._service}
               .configValue=${"service"}
               @value-changed=${this._valueChanged}
             ></ha-service-picker>
-            <b>
-              ${this.hass!.localize(
-                "ui.panel.lovelace.editor.action-editor.editor_service_data"
-              )}
-            </b>
+            ${this._service
+              ? html`<div class="secondary">
+                    ${computeServiceDescription(
+                      this.hass,
+                      this._service.split(".", 2)[0],
+                      this._service.split(".", 2)[1]
+                    )}
+                  </div>
+                  <div class="service-data-editor">
+                    <ha-yaml-editor
+                      .defaultValue=${this._service_data}
+                      .configValue=${"service_data"}
+                      @value-changed=${this._valueChanged}
+                    ></ha-yaml-editor>
+                  </div>
+                  <table class="attributes">
+                    <tr>
+                      <th>
+                        Parameter - Translate
```

Comment:
Does that mean you still have to translate it? :smile:

Reply:
Yes :rofl: